### PR TITLE
Capstone branch

### PR DESCRIPTION
**Pull request by ohyajk**
I added HTML, CSS and Js for the First Capstone Project in Microverse course. I made a conference website about a travel event using Tailwind CSS Framework.

You can view Deployment here :-

[Github Demo Link](https://ohyajk.github.io/Capstone-1/)

You can view Loom video here :-

[Loom Video Link](https://www.loom.com/share/363627929d594495b05220ad56f88100)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+<!-- FOOTER SECTION -->
+
+<footer class="flex w-full justify-center p-2 gap-2 items-center my-8">
+    <div class="flex justify-end w-1/2">
+    <img class="h-20 object-contain" src="./Assets/partners/CreativeCommons_logo_trademark.svg" alt="cc logo">
+    </div>
+    <div class="w-1/2">
+    <h2 class="hover:text-[#f28f10]">© All RIGHTS RESERVED 2022</h2>
+    </div>
+</footer>
```
Comment (Santiago220991) [inline index.html]: Kindly make sure that your project sticks as much as possible to the template provided. The desktop footer section in your project is different from the template provided. You could use Lorem ipsum text to generate the missing text.
Your design
image

Template design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+<!-- FOOTER SECTION -->
+
+<footer class="flex w-full justify-center p-2 gap-2 items-center my-8">
+    <div class="flex justify-end w-1/2">
+    <img class="h-20 object-contain" src="./Assets/partners/CreativeCommons_logo_trademark.svg" alt="cc logo">
+    </div>
+    <div class="w-1/2">
+    <h2 class="hover:text-[#f28f10]">© All RIGHTS RESERVED 2022</h2>
+    </div>
+</footer>
```
Comment (Santiago220991) [inline about.html]: The footer section on the about page should be different from the main page. Kindly change the background color and please add the missing text to make your project stick as much as possible to the template provided. :+1: :smiley: :smiley: 
Your design
image

Template design
image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
```
Comment (Santiago220991) [inline about.html]: Kindly remove this section from this page. This section doesn't appear on the template about page. 
Your design
image

Template design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+                </a>
+
+                <a class="block md:hidden" href="#">
+                    <button type="button" class="my-14 text-lg text-white bg-[#f28f10] p-4 active:border-white active:border-2 ">
+                        JOIN PROGRAM
+                    </button>
+                </a>
+            
+
+        </section>
```
Comment (Santiago220991) [inline index.html]: Kindly make sure that your project sticks as much as possible to the template provided. :eyes: :eyes: 
- Desktop version: The space between cards is different from the template.

Your design
image

Template provided
image

- Mobile version: The space between cards is different from the template, and the cards should have the same high.

Your design
image

Template design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
```
Comment (Santiago220991) [inline index.html]: In your mobile version, this section should not appear on the main page. It appears just on the about page. Kindly fix this. :pray: :pray: 
Your design
image

Template design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @ohyajk, 
I'm Santiago your reviewer for this PR.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Good commit messages.

## Required Changes ♻️
- Kindly update your README file. Last week Microverse provided a new template that must be used in every project. :briefcase: :briefcase:  Also please don't forget to give appropriate credit to Cindy Shin the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgments section.

- As part of the requirements of the project you have to record a presentation video for your project. Kindly include this in your PR summary. :pray: :pray: Next reviewer is going to check the video and could ask for changes if needed. 
image

- Most of the texts, borders, titles, and subtitles are using colors that not are provided by the template. Please make sure that you are using the same basic colors provided by the template. #272A31, #EC5242, #D3D3D3 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (ohyajk) [reply index.html]: can you please give name of section which I have to remove. Your this and that I am not able to understand. @Santiago220991

**Review by Lameck1 — COMMENTED** [review]
Hello @ohyajk👋,

_This is an additional review as a result of the second opinion request._

One of your complaints is that @Santiago220991 did not give you the link to the new README file template. This means that you are not keeping up with the announcements in the Microverse Students slack workspace. Information about the new README file template was given out to students last week by Microverse. Nonetheless, here is the [LINK](https://github.com/microverseinc/readme-template) to Microverse's README template.

The other complaint is about @Santiago220991 not being clear in the suggested changes, especially, in the last inline comment. I don't see what's not clear about @Santiago220991's comments. @Santiago220991 has clearly highlighted the lines of code that have issues you need to address. So, when @Santiago220991 says `this`, it means the `<section>` highlighted by the inline comment. @Santiago220991 has even gone ahead and included a screenshot of your implementation `vs` the provided design for you to be able to realize the differences between the two and make the necessary adjustments. @Santiago220991 has done a really good job in guiding you here.

Please on every concern from @Santiago220991's review and then ask for another review when done. All the best.

Keep rocking 🚀!

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ohyajk 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✔️ Followed GitHub flow.
✔️ Good commit messages.
✔️ Nice implementation of 'show more speakers' feature.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**